rubin_scheduler/utils/consdb.py

```diff
@@ -867,59 +890,58 @@ def opsim(self) -> pd.DataFrame:
-                "azimuth": self.azimuth,
-                "paraAngle": self.parallactic_angle,
-                "psudoParaAngle": self.pseudo_parallactic_angle,
-                "cloud": self.cloud,
-                "moonAlt": self.sun_moon_positions["moon_alt"],
-                "sunAlt": self.sun_moon_positions["sun_alt"],
-                "note": self.note,
-                "target": self.target_name,
-                #                "fieldId": None,
-                #                "proposalId": None,
-                #                "block_id": None,
-                "observationStartLST": self.observation_start_lst,
-                "rotTelPos": self.rot_tel_pos,
-                "rotTelPos_backup": None,
-                "moonAz": self.sun_moon_positions["moon_az"],
-                "sunAz": self.sun_moon_positions["sun_az"],
-                "sunRA": self.sun_moon_positions["sun_RA"],
-                "sunDec": self.sun_moon_positions["sun_dec"],
-                "moonRA": self.sun_moon_positions["moon_RA"],
-                "moonDec": self.sun_moon_positions["moon_dec"],
-                "moonDistance": self.moon_distance,
-                "solarElong": self.solar_elong,
-                "moonPhase": self.sun_moon_positions["moon_phase"],
-                #                "cummTelAz": None,
-                #                "scripted_id": None,
-                "start_date": self.consdb_visits["obs_start"],
-                "t_eff": self.consdb_visits["eff_time_median"],
-                "seq_num": self.seq_num,
-            }
-        )
+        opsim_dict = {
+            "observationId": self.observation_id,
+            "fieldRA": self.ra,
+            "fieldDec": self.decl,
+            "observationStartMJD": self.obs_start_mjd,
+            "flush_by_mjd": np.nan,
+            "visitExposureTime": self.shut_time,
+            "filter": self.band,
+            "rotSkyPos": self.sky_rotation,
+            "rotSkyPos_desired": np.nan,
+            "numExposures": self.num_exposures,
+            "airmass": self.airmass,
+            "seeingFwhm500": self.fwhm_500,
+            "seeingFwhmEff": self.fwhm_eff,
+            "seeingFwhmGeom": self.fwhm_geom,
+            "skyBrightness": self.sky_mag_per_asec,
+            "night": self.night,
+            "slewTime": np.nan,
+            "visitTime": self.visit_time,
+            "slewDistance": self.slew_distance,
+            "fiveSigmaDepth": self.eff_time_m5,
+            "altitude": self.altitude,
+            "azimuth": self.azimuth,
+            "paraAngle": self.parallactic_angle,
+            "psudoParaAngle": self.pseudo_parallactic_angle,
+            "cloud": self.cloud,
+            "moonAlt": self.sun_moon_positions["moon_alt"],
+            "sunAlt": self.sun_moon_positions["sun_alt"],
+            "note": self.note,
+            "scheduler_note": "NOT AVAILABLE",
+            "target_name": self.target_name,
+            #            "block_id": "NOT AVAILABLE",
```

Comment:
We may want to pass along block_id, as it's a valid thing in the consdb .. but we do need to think about what it means I suppose, given we have a field called this in the opsim outputs which is not the same thing as it is in the consdb outputs.